default/stringtables/en.txt

```diff
@@ -13340,7 +13340,7 @@ EXOBOT_SHORT_DESC
 Builds Exobot Citizens
 
 STARLANE_SPEED_SHORT_DESC
-Increases Starlane Speed
+Unlocks Logistics Focus
```

Comment:
Does `Logistics Focus` has it's one page, could we make a reference here?

Reply:
It does.
![logisticsfocus](https://user-images.githubusercontent.com/43386669/178136149-b8a6664b-05e6-49d0-a232-10d69f63fcc6.png)
It's also in the actual description for the tech:
![InterstellarLogistics](https://user-images.githubusercontent.com/43386669/178136188-ffca56a8-4f14-47b1-9f20-e3b50d858a4b.png)

Reply:
Could you replace "Logistics Focus" with the reference 
```
[[encyclopedia LOGISTICS_FOCUS_TITLE]]
```

it's better to keep things linked.  Also, this will also help translates, they often do bulk updates and might miss that this term should by synced with other entries.

Reply:
I can though changing the short description does not create a link:
![interstellar](https://user-images.githubusercontent.com/43386669/178138452-32c3feb0-1ce6-4072-9571-488646b95b3f.png)

I went ahead and changed it where it appears in the body of the text

Reply:
> I can though changing the short description does not create a link:

Does it insert the name of the referenced article there?

Reply:
It looks exactly like in the screenshot. The correct name, but no hyperlink

Reply:
I think this is expected behaviour. Not all UI displayed text is "rich text" which would create clickable links.

Reply:
So was the end decision here that I should use a link anyway? Even though it's not rich text?

Apart from that this is done, I just need to test everything works correctly and clean up the commits a bit.

Reply:
But this will insert the proper text from the reference.  Even if it's not a link, it still requires less maintenance from the translators